config.py

```diff
@@ -1063,6 +1063,12 @@ class ArbitraryContrast(TypedDict):
 on individual EOG epochs.
 """
 
+ssp_meg: str = 'separate'
+"""
+Whether to compute SSP vectors for MEG channels separately (`'separate'`)
+or jointly (`'combined'`) for magnetometers and gradiomenters.
```

Comment:
can you add a note here for why one option is better than another? I have to admit I never looked into this. Should we say it's recommended to use "combined" when using maxfilter?

Reply:
Because mag and grad signals are jointly created/synthesized by MaxFilter (they are not independent afterward) it makes sense mathematically to estimate the projectors from them jointly

Reply:
If the case is so clear-cut, can we default to None here and set it to combine if the user requested Maxwell-filtering, and separate otherwise?

Reply:
This brings me back to my top comment:

> When using maxfiltered data, meg='combined' usually makes sense. Maybe we should have this default to True meaning "combined when using maxfilter and separate otherwise", but this should maybe be an upstream change first in MNE-Python. In the meantime, let's expose the option at least.

I think @agramfort was never totally convinced of the utility here, so I'm hesitant to automatically enable it for people

Reply:
> This brings me back to my top comment:

Oh, I missed that one, sorry.